templates/barbican/config/barbican-p11-prep-config.json

```diff
@@ -13,14 +13,16 @@
 	"owner": "barbican",
 	"perm": "0600"
       },
+      {{- if (index . "P11ClientDataPath") }}
       {
-        "source": "/var/lib/config-data/default/Chrystoki.conf",
-        "dest": "/usr/local/luna/Chrystoki.conf",
+        "source": "/etc/p11-client/*",
```

Comment:
should be /var/lib/config-data/hsm, right ?

Reply:
Done